docs/source/autoscaling.rst

```diff
@@ -27,11 +27,12 @@ a situation where there is a loss of autoscaling state, a service up is the
 safest course of action until the autoscaler can scale back down to a "normal"
 instance count.
 
-Autoscaling only considers marathon tasks that have passing health checks. This
-is so new tasks that have higher-than-average load when starting up are ignored
-until their first health check passes. PaaSTA will not autoscale a service
-without any health checks, though these services can be autoscaled using the
-``bespoke`` decision policy.
+If you use autoscaling it is highly recommended that you also configure a
```

Comment:
Yup, interesting background for me anyway. Yesterday I actually tried creating without an app without a healthcheck just to try this out.